sql/core/src/test/scala/org/apache/spark/sql/MathFunctionsSuite.scala

```diff
@@ -262,6 +262,14 @@ class MathFunctionsSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("SPARK-44973 conv must allocate enough space for all digits plus negative sign") {
```

Comment:
nit.
```
- SPARK-44973
+ SPARK-44973:
```

Reply:
I was following the convention in the file 🤷